content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -241,22 +241,22 @@ waypoint default/bookinfo-productpage applied
 View the `productpage` waypoint proxy status; you should see the details of the gateway
 resource with `Ready` status:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=none >}}
```

Comment:
No longer needed since test is now using Test is now using `_wait_for_gateway`.

Reply:
Since the primary purpose of doc tests (with side benefit of testing Istio)  is to test that the documented instructions work as documented, a general best practice is to call snips wherever we can. The util functions are intended for docs that don't include explicit instructions, e.g., docs that just say before-you-begin install bookinfo, or something like that.